### ISSUE TITLE
ASIM Sysmon parser doesn't parse Process Create events properly from the Event table

### ISSUE THREAD

**Issue by szabolevo** [Azure/Azure-Sentinel#5716]
**Describe the bug**
ASIM Sysmon parser **vimProcessCreateMicrosoftSysmon** doesn't parse Process Create events properly from the **Event** table. The Process Create Events are not returned when  `vimProcessCreateMicrosoftSysmon` table is used to get these logs. 

**To Reproduce**
1. Go to 'Microsoft Sentinel', ensure Sysmon logs are ingested for at least 1 client (ensure that no other sources provide the same logs)
2. Once logs show up in Sentinel run the below query:
 `vimProcessCreateMicrosoftSysmon` | take 100
3. The response is `No results found from the specified time range`

**Expected behavior**
Query should parse and return the ingested logs from Sysmon querying `vimProcessCreateMicrosoftSysmon` table

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser: Firefox
 - Version: 102
**Additional context**

**Comment by oshezaf** [comment]
Hi @szabolevo : Thanks for fixing. 

I am in charge of the ASIM parser, for which the change is minor and very important. To remove the dependency on the review of the more complex changes in the other parser, I will create a separate PR for that.

**Comment by oshezaf** [comment]
Hi @szabolevo : Thanks for fixing! 

I am in charge of the ASIM parser, for which the change is minor and very important. I will create a separate PR for it, so it is not blocked on the review of the other Sysmon parsers.

**Comment by oshezaf** [comment]
Here is the PR: https://github.com/Azure/Azure-Sentinel/pull/5721
